h2quic/client_test.go

```diff
@@ -44,19 +46,31 @@ var _ = Describe("Client", func() {
 
 	It("saves the TLS config", func() {
 		tlsConf := &tls.Config{InsecureSkipVerify: true}
-		client = newClient(tlsConf, "", &roundTripperOpts{})
+		client = newClient("", tlsConf, &roundTripperOpts{}, nil)
 		Expect(client.tlsConf).To(Equal(tlsConf))
 	})
 
+	It("saves the QUIC config", func() {
+		quicConf := &quic.Config{HandshakeTimeout: time.Nanosecond}
+		client = newClient("", &tls.Config{}, &roundTripperOpts{}, quicConf)
+		Expect(client.config).To(Equal(quicConf))
+	})
+
+	It("uses the default QUIC config if none is give", func() {
+		client = newClient("", &tls.Config{}, &roundTripperOpts{}, nil)
+		Expect(client.config).ToNot(BeNil())
+		Expect(client.config).To(Equal(defaultQuicConfig))
+	})
+
 	It("adds the port to the hostname, if none is given", func() {
-		client = newClient(nil, "quic.clemente.io", &roundTripperOpts{})
+		client = newClient("quic.clemente.io", nil, &roundTripperOpts{}, nil)
 		Expect(client.hostname).To(Equal("quic.clemente.io:443"))
 	})
 
 	It("dials", func(done Done) {
-		client = newClient(nil, "localhost:1337", &roundTripperOpts{})
+		client = newClient("localhost:1337", nil, &roundTripperOpts{}, nil)
 		session.streamsToOpen = []quic.Stream{newMockStream(3), newMockStream(5)}
-		client.dialAddr = func(hostname string, _ *tls.Config, _ *quic.Config) (quic.Session, error) {
+		dialAddr = func(hostname string, _ *tls.Config, _ *quic.Config) (quic.Session, error) {
```

Comment:
Again, bc github hides it: The var should be reset to the original value after each test.